lightningd/test/run-invoice-select-inchan.c

```diff
@@ -6,14 +6,6 @@
 bool deprecated_apis = false;
 
 /* AUTOGENERATED MOCKS START */
-/* Generated stub for bitcoind_gettxout */
```

Comment:
I was about to say that this commit needs to be squashed into the previous one, to maintain bisectability. However it just removes a bunch of definitions, so I expect this to be a left-over from some other refactor.